.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "Traffic dev env",
+	"image": "ghcr.io/xoolive/traffic/dev:latest",
+	// "build": {
```

Comment:
I am not sure what's best here, in a way, it's nice to have the Docker image ready, but on the other hand, if we update `poetry.Dockerfile` the thing will break, no? Wouldn't it be better to with the "build" option?

Reply:
Ideally, each time you change something in the dependencies and you push it, you'd create a new image as well. It is then up to the user to keep the dev container up-to-date.